packages/zowe-explorer/src/trees/dataset/DatasetFSProvider.ts

```diff
@@ -203,17 +203,30 @@ export class DatasetFSProvider extends BaseProvider implements vscode.FileSystem
         const uriPath = uri.path.substring(uriInfo.slashAfterProfilePos + 1).split("/");
         const pdsMember = uriPath.length === 2;
         if (!entryExists) {
-            const resp = await ZoweExplorerApiRegister.getMvsApi(uriInfo.profile).dataSet(entryIsDir ? uriPath[0] : path.parse(uriPath[0]).name, {
-                attributes: true,
-            });
-            if (resp.success) {
-                const dsorg: string = resp.apiResponse?.items?.[0]?.dsorg;
-                entryIsDir = pdsMember ? false : dsorg?.startsWith("PO") ?? false;
+            if (pdsMember) {
+                const resp = await ZoweExplorerApiRegister.getMvsApi(uriInfo.profile).allMembers(uriPath[0], {
+                    attributes: true,
+                });
```

Comment:
For members I don't believe attributes are needed since we only look at names on the API response. Listing without attributes may make the fetch operation more efficient:

```suggestion
                const resp = await ZoweExplorerApiRegister.getMvsApi(uriInfo.profile).allMembers(uriPath[0]);
```